src/timeline/TimelineEventsVisualizer.jsx

```diff
@@ -60,6 +59,27 @@ class TimelineEventsVisualizer extends Visualizer {
         };
     };
 
+    componentDidMount() {
+        window.addEventListener('load', this.handleLoad);
+    }
+
+    componentDidUpdate() {
+        if (!this.stickySupport) {
+            this.updateMedicationItems();
```

Comment:
[DP2] We should use lodash to throttle this somewhat. The performace is super choppy on my IE instance, and I anticipate it'll be marginally better with a throttled call here (by something like 300ms ish?)

Reply:
throttling it will make the text jump, it will only move once every 300ms.  Is that what we want?